src/main/java/io/ebean/migration/MigrationVersion.java

```diff
@@ -111,7 +113,7 @@ public String nextVersion() {
    */
   private String formattedVersion(boolean normalised, boolean nextVersion) {
 
-    if (ordering == REPEAT_ORDERING) {
+    if (isRepeatable()) {
       return "R";
```

Comment:
Do we want to return "I" or "R" here ?  I guess not, will check.

Reply:
Ah looks all good. LocalMigrationResource.key() ... has a pre check.  Maybe this area needs a tidy up in future
... but this change is all good.